src/world/area_kkj/kkj_25/B06D10.c

```diff
@@ -1,35 +1,176 @@
 #include "kkj_25.h"
+#include "effects.h"
 
 #include "world/common/UnkFoldFunc.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_25/B06D10", func_802406EC_B070EC);
+ApiStatus func_802406EC_B070EC(Evt* script, s32 isInitialCall) {
+    Bytecode* args = script->ptrReadPos;
+    s32 spriteIndex = evt_get_variable(script, *args++);
+    s32 rasterIndex = evt_get_variable(script, *args++);
+    UnkEntityStruct* unkStruct = heap_malloc(sizeof(*unkStruct));
```

Comment:
Based on a similar file and on the struct size, I assumed this to be an `UnkEntityStruct`, but I can't tell for sure. Would it be preferable to just create another `UnkStruct`?

Reply:
Looking at the fields set below this is almost certainly a correct assumption, I say keep UnkEntityStruct 👍